README.fr.md

```diff
@@ -217,7 +217,7 @@ Voir le [dépôt dédié à ces extensions](https://github.com/FreshRSS/Extensio
 
 # APIs et applications natives
 
-FreshRSS supporte l’accès depuis des applications natives pour Linux, Android, iOS, et OS X, grâce à deux APIs distinctes :
+FreshRSS supporte l’accès depuis des applications natives pour Linux, Android, iOS, Windows et MacOS, grâce à deux APIs distinctes :
```

Comment:
Or maybe `macOS`? (To be clear, I don't care.)

Reply:
Oh, you are right.